data/bangs.json

```diff
@@ -4158,7 +4158,7 @@
     "s": "Kagi Search",
     "d": "kagi.com",
     "t": "analyser",
-    "u": "/search?q={{{s}}}+site%3Ahttp%3A//larmarange.github.io/analyse-R",
+    "u": "/search?q={{{s}}}+site:http://larmarange.github.io/analyse-R",
```

Comment:
This should be https

Reply:
Applied all review comments up to here; changes pushed this PR.